src/components/DivElement.js

```diff
@@ -1,6 +1,13 @@
 const HTMLElement = require('./HTMLElement');
 
 // Define class here
+class DivElement extends HTMLElement {
+  constructor(content) {
+    super(content);
+    this.tag = 'div';
+    this.content = content;
```

Comment:
While this works, I would refrain from explicitly defining `this.tag` and `this.content` in this scenario. Reason being is that we can offload that work to the `HTMLElement` constructor.

There can be scenarios where we start editing the `HTMLElement` constructor, expecting those changes to be applied to child classes. Here's an example: I decide to alter all HTMLElements so that the tag is always set to lowercase:

```js
class HTMLElement {
  constructor(tag, content) {
    this.tag = tag.toLowerCase();
    this.content = content;
  }

  ...
}
```

Instead, call the parent constructor with `super()`, and let it take care of `tag` and `content` for you 😃. Here are some more examples: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/super